kolibri/plugins/device/assets/src/views/FacilitiesPage/FacilitiesTasksPage.vue

```diff
@@ -88,10 +87,9 @@
       },
     },
     $trs: {
-      backToFacilitiesLabel: {
-        message: 'Back to facilities',
-        context:
-          'Link to take user back to the Device > Facilities page where they can see the list of facilities.',
+      facilitiesTaskManagerTitle: {
```

Comment:
This is removing the previous string that was a `basic-link` style link with an arrow like `<- Back to Facilities` because this page has changed from being an AppBar page to being an Immersive page -- this is to keep it consistent with the other content task manager page. There is no basic link in that design since the immersive page has the back arrow.

I added a string "Facilities - Task Manager" to give the immersive page a clear name that distinguishes it from the other separate task manager.

Reply:
Sorry - I should have posted screen shots in the PR description